parsec/scheduling.c

```diff
@@ -411,6 +414,8 @@ int __parsec_complete_execution( parsec_execution_stream_t *es,
             parsec_taskpool_update_runtime_nbtask(tp, -1);
     }
 
+    PARSEC_PINS(es, COMPLETE_EXEC_END, task);
```

Comment:
This event was meant to compute the user side completion of a task, excluding anything related to runtime.